core/src/main/java/com/radixdlt/keys/PersistedBFTKeyManager.java

```diff
@@ -77,8 +77,18 @@
 public final class PersistedBFTKeyManager {
   private final ECKeyPair ecKeyPair;
 
-  public PersistedBFTKeyManager(String nodeKeyPath) {
-    this.ecKeyPair = loadNodeKey(nodeKeyPath);
+  public PersistedBFTKeyManager(String nodeKeyPath, boolean createNodeKeyIfMissing) {
+    ECKeyPair nodeEcKeyPair;
+    try {
+      nodeEcKeyPair = loadNodeKey(nodeKeyPath);
```

Comment:
I don't think is ideal - if this node restarts, its key will be lost - which I don't think we want.

Instead, one of the `Keys.readKey` takes a `create` parameter - which creates a key at the given location - we should be using that.

That way the key will be persisted across docker image restarts.

Reply:
> I don't think is ideal - if this node restarts, its key will be lost - which I don't think we want.
I am viewing from the perspective where anything using private keys should be intentional and explicitly manage by the user - i.e. you want (or need?) your private key to persist - create and manage it yourself :). I don't want people to rely on a private key generated inside the docker.

This is a mere convenience functionality (I don't really care about details, just give me a quick way to scoop in the network).